bin/reth/src/config.rs

```diff
@@ -0,0 +1,16 @@
+//! CLI command to write default config to stdout
+use clap::Parser;
+use reth_staged_sync::Config;
+
+/// `reth config` command
+#[derive(Debug, Parser)]
+pub struct Command;
+
+impl Command {
+    /// Execute `config` command - write default config to stdout
+    pub async fn execute(&self) -> eyre::Result<()> {
+        let config = Config::default();
```

Comment:
need to load the active config here, same as:

https://github.com/paradigmxyz/reth/blob/087d0a63170b944dc70435c691ec54069ccfeea2/bin/reth/src/node/mod.rs#L404-L404